### PR TITLE
RemoteMessagingFramework component

### PR DESCRIPTION
**Pull request by vkraucunas**
**Asana Task/Github Issue:** 
[Link to Asana Task/Github Issue](https://app.asana.com/0/1208403786889309/1208398379851629/f)

## Description

- Adds a RemoteMessagingFramework component to the New Tab Page

## Testing Steps

- 

## Checklist

*Please tick all that apply:*

- [x] I have tested this change locally
- [ ] I have tested this change locally in all supported browsers
- [ ] This change will be visible to users
- [x] I have added automated tests that cover this change
- [ ] I have ensured the change is gated by config
- [ ] This change was covered by a ship review
- [ ] This change was covered by a tech design
- [ ] Any dependent config has been merged

### CODE REVIEW

**Comment by shakyShane** [comment]
* **#1140** <a href="https://app.graphite.dev/github/pr/duckduckgo/content-scope-scripts/1140?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#1138** <a href="https://app.graphite.dev/github/pr/duckduckgo/content-scope-scripts/1138?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#1114** <a href="https://app.graphite.dev/github/pr/duckduckgo/content-scope-scripts/1114?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @vkraucunas and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Review by shakyShane — CHANGES_REQUESTED** [review]

File: special-pages/pages/new-tab/app/remote-messaging-framework/RemoteMessagingFramework.module.css [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+    &:disabled {
+        color: var(--color-black-at-84);
+    }
+
+    &:disabled:hover {
+        cursor: not-allowed;
+        background-color: var(--color-white-at-6);
+    }
+
+    &:focus-visible {
```
[the remark this answers is not in the document]
Comment (vkraucunas) [reply special-pages/pages/new-tab/app/remote-messaging-framework/RemoteMessagingFramework.module.css:91]: Moved to a box-shadow, like the Figma

File: special-pages/pages/new-tab/app/remote-messaging-framework/RemoteMessagingFramework.module.css [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+        outline: 2px solid var(--ddg-color-primary);
+        border: 1px solid white;
+    }
```
[the remark this answers is not in the document]
Comment (vkraucunas) [reply special-pages/pages/new-tab/app/remote-messaging-framework/RemoteMessagingFramework.module.css]: added the `--ntp-focus-outline-color` to replace the blue values in the box shadow

File: special-pages/pages/new-tab/app/remote-messaging-framework/RemoteMessagingFramework.module.css [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+    padding-right: var(--sp-3);
+    height: var(--sp-8);
+    background-color: var(--color-black-at-6);
+    border-width: 0;
+    border-radius: 6px;
+    text-wrap: nowrap;
+
+    * {
+        user-select: none;
+    }
```
[the remark this answers is not in the document]
Comment (vkraucunas) [reply special-pages/pages/new-tab/app/remote-messaging-framework/RemoteMessagingFramework.module.css]: removed

File: special-pages/pages/new-tab/app/remote-messaging-framework/RemoteMessagingFramework.module.css [hunk 4]
```diff
@@ -0,0 +1,195 @@
...
+
+.title {
+    font-weight: var(--title-2-font-weight);
+    line-height: normal;
+    margin-bottom: var(--sp-2);
+}
+
+.description {
+    line-height: 1.25rem;
+    margin-bottom: var(--sp-3);
```
Comment (shakyShane) [inline special-pages/pages/new-tab/app/remote-messaging-framework/RemoteMessagingFramework.module.css:43]: I think the margin-bottom curse has found us! the small+medium layouts look affected https://deploy-preview-1114--content-scope-scripts.netlify.app/build/pages/new-tab/?rmf=small

**Review by shakyShane — CHANGES_REQUESTED** [review]

File: special-pages/pages/new-tab/app/remote-messaging-framework/RemoteMessagingFramework.module.css (hunk 4 above)
Comment (vkraucunas) [reply special-pages/pages/new-tab/app/remote-messaging-framework/RemoteMessagingFramework.module.css]: fix pushed

**Review by shakyShane — APPROVED** [review]
Thanks @vkraucunas.

Since this wont be used just yet (but has been design reviewed), let's merge it now and follow up with docs (to keep the PR smaller)

**Pull request merged**